pkg/controller/command/verifiable/command.go

```diff
@@ -711,16 +711,24 @@ func (o *Command) GeneratePresentationByID(rw io.Writer, req io.Reader) command.
 		return command.NewValidationError(GeneratePresentationByIDErrorCode, fmt.Errorf("get vc by id : %w", err))
 	}
 
-	doc, err := o.didStore.GetDID(request.DID)
+	var didDoc *did.Doc
```

Comment:
I think it should be other way around. If someone caches DID then local storage should be looked first.

Reply:
Thank you for sharing your good opinion. If the DID is cached, it has been modified to look in the local storage first.

I have an additional question.
When I look into the parts of DID search in many protocols, I look into the VDR Registry first, and if not, look into local storage. (I also referred to that part.)
Does this need to be modified?

Reply:
@msjeong482 looks good to me now. Please rebase so that we can merge this PR.